### ISSUE TITLE
Decode Urls in xhr requests / responses

### ISSUE THREAD

**Issue by jennifer-mann** [cypress-io/cypress#75]
The urls are being encoded since release `0.12.0` so that my RegExp matching is now failing. Can you decode urls automatically or allow a key for encoded vs decoded urls. 

**Example Code:**
```coffeescript
  it "fetches accounts ordered by 'name asc' by default [zmv]", ->
    cy
       .server()
       .route(/accounts/, "fixture:accounts").as("accountsGet")
       .wait("@accountsGet").its("url").should("match", /orderby=name\+asc/)
```

**Example Error:**
```
CypressError: 
  Timed out retrying: 
    expected 'http://localhost:2020/accounts?page=1&$filter=(rowStatus%2Beq%2B1)&$orderby=name%2Basc&includeOpenFoldersCount=true&includeStatusCount=true' to match /orderby=name\+asc/
```

**Comment by brian-mann** [comment]
Fixed in [`0.12.1`](https://github.com/cypress-io/cypress/wiki/Changelog#0121-10282015).

**Comment by jennifer-mann** [comment]
This is still failing. I'm not sure if the fix was not in `0.12.1` or if perhaps my update didn't take for some reason?  Or perhaps it has to do with my using `.its("url")`?

**Error in 0.12.1**
```
expected 'http://localhost:2020/accounts?page=1&$filter=(rowStatus%2Beq%2B1)&$orderby=name%2Basc&includeOpenFoldersCount=true&includeStatusCount=true' to match /orderby=name\+asc/
```
